### PR TITLE
cult tweaks 

### PR DESCRIPTION
**Pull request by ShadowDeath6**
I didn't test my code again >:^)

:cl: ShadowDeath6
rscadd: Cooldown on summoning spirits into soulstones via manifest rune.
rscadd: Adds description information on pylons and shielded robes. You could recharge robes?
rscadd: Cults can no longer use the manifest rune while there are more living constructs than living cultists.
/:cl:

EDIT: changelog
-Cruix

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
oh no

File: code/game/gamemodes/cult/runes.dm [hunk 1]
```diff
@@ -911,12 +913,25 @@ var/list/teleport_runes = list()
...
 	for(var/obj/item/device/soulstone/S in get_turf(src))
 		user <<"<span class='warning'>You attempt to absorb the manifested soul of [ghost_to_spawn] through [S]!</span>"
+		if(manifest_construct_cooldown > world.time)
+			user <<"<span class='warning'>It's been too long since a spirit has been manifested into a stone. Wait [manifest_construct_cooldown - world.time] seconds.</span>"
+			return
+		for(var/mob/living/simple_animal/hostile/construct/C in mob_list) //don't need a living check because constructs gib when they die
+			if(iscultist(C))
+				living_constructs++
+		for(var/mob/living/carbon/human/H in mob_list)
+			if(iscultist(H) && H.stat != DEAD) // living cultists
```
Comment (ShadowDeath6) [inline code/game/gamemodes/cult/runes.dm]: it fucks up here for a future reminder to myself

**Comment by ghost** [comment]
question, are cult structures actually breakable

**Comment by ShadowDeath6** [comment]
If they were made breakable by that object damage refactor then no

**Comment by ShadowDeath6** [comment]
hold on I have another thing

**Pull request merged**